src/main/java/com/spotify/reaper/cassandra/JmxProxy.java

```diff
@@ -285,6 +290,14 @@ public int getPendingCompactions() {
    * @return true if any repairs are running on the node.
    */
   public boolean isRepairRunning() {
+    return isRepairRunningPre22() || isRepairRunningPost22() || isValidationCompactionRunning();
```

Comment:
@nbozic, since we can detect each repair session individually by listing the Repair#* mbeans, we should be fine this way. It replaces the way it was done with AntiEntropySessions pre-2.2.

We'll add another test if we see that it's not enough, but I think we won't need it.